addon/models/draft-registration.js

```diff
@@ -1,12 +1,22 @@
 import DS from 'ember-data';
 import OsfModel from './osf-model';
 
+/**
+ * Model for OSF APIv2 draft registrations.
+ * This model represents draft registration data - the initiator of the draft,
```

Comment:
Our API has some slightly unusual behavior, in that certain models/collections can only be accessed via relationships (`this.store.find('modelname')` would fail).

To help avoid confusion, can you please indicate in the text whether this is intended for direct querying, relationships, or both?

For maintainability reasons, we're trying to keep the list of fields/descriptions all in one place (the swagger docs). If we do want to explain fields in more detail, we should do so by calling out each field (and type) explicitly in [YUIDoc](http://yui.github.io/yuidoc/syntax/) syntax.